src/Kunstmaan/AdminListBundle/Service/ExportService.php

```diff
@@ -92,7 +92,7 @@ protected function streamOutput(ExportableInterface $adminList, $format)
             $row = [];
             /** @var Field $column */
             foreach ($adminList->getExportColumns() as $column) {
-                $row[] = $this->translator->trans($column->getHeader());
+                $row[] = $this->translator->trans($column->getHeader(), [], null, 'en');
```

Comment:
Why is this change needed?

Reply:
because the importer logic is hardcoded to english. on projects where your admin has an adminlocale different to english the exported file will have the headers domain/keyword/status translated. When importing the same excel file after lets say adding some new translations the cms will respond saying the excel file is invalid because it does not find the required headers domain/keyword/status

Reply:
By making it so that the excel file exported will contain the headers standardized in english the export / import flow will be garuanteed to stay working